### PR TITLE
Update Harvester v1.0 install configs

### PR DESCRIPTION
**Pull request by guangbochen**
- Update Harvester v1.0 install configs
- Update Rancher usage to v2.6.3

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -77,26 +77,29 @@ During the installation, you can either choose to form a new cluster or join the
...
-1. Choose the installation device to which the Harvester cluster will be formatted.
+1. Choose the installation device to which the Harvester cluster will be formatted
+     - Note: you will be promoted to select the [MBR](https://en.wikipedia.org/wiki/Master_boot_record) disk partition when install via the BIOS method, otherwise [GPT](https://en.wikipedia.org/wiki/GUID_Partition_Table) will be used by default.
+   ![iso-install-disk.png](./install/assets/iso-install-disk.png)
 1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bond NIC named `harvester-mgmt`, and the IP address can either be configured via DHCP or by static method.
    ![iso-installed.png](./install/assets/iso-nic-config.gif)
 1. Optional: Configure the DNS servers; use commas as delimiters.
-1. Configure the Virtual IP with which you can use to access the cluster or join other nodes to the cluster.
+1. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster <small>(Note: You will need to config static MAC-to-IP address mapping on your DHCP server to persist the Virtual IP accordingly)</small>.
```
Comment (gitlawr) [inline docs/index.md:86]: s/config/configure/ , also check other places

File: docs/index.md [hunk 2]
```diff
@@ -77,26 +77,29 @@ During the installation, you can either choose to form a new cluster or join the
...
-1. Configure the login password of the host. The default ssh user is `rancher`.
-1. Optional: Configure the NTP Servers of the node if needed. Default is `0.suse.pool.ntp.org`.
-1. Optional: If you need to use an HTTP proxy to access the outside world, enter the proxy URL address; otherwise, leave this blank.
-1. Optional: You can choose to import SSH keys from a remote URL server. Your GitHub public keys can be used with `https://github.com/<username>.keys`.
-1. Optional: If you need to customize the host with cloud-init configuration, enter the HTTP URL.
+1. Configure the login password of the host. The default SSH user is `rancher`.
+1. (Optional) Configure the NTP Servers of the node. This defaults to `0.suse.pool.ntp.org`.
+1. (Optional) If you need to use an HTTP proxy to access the outside world, enter the proxy URL address here. Otherwise, leave this blank.
+1. (Optional) You can choose to import SSH keys from a remote URL server. Your GitHub public keys can be used with `https://github.com/<username>.keys`.
+1. (Optional) If you need to customize the host with a cloud-init config, enter the HTTP URL here.
```
Comment (gitlawr) [inline docs/index.md:92]: `a cloud-init config` => `[Harvester configuration](./install/harvester-configuration.md)`

File: docs/install/usb-install.md [hunk 3]
```diff
@@ -49,4 +47,4 @@ You can press `e` to edit the GRUB menu entry and append `nomodeset` to the boot
     If a USB flash driver boots, but you can't see the harvester installer. You may try out the following workarounds:
     
     - Plug the USB flash drive into a USB 2.0 slot.
-    - For version `v0.3.0`, try to remove the `console=ttyS0` parameter when booting. You can press `e` to edit the GRUB menu entry and remove the `console=ttyS0` parameter.
+    - For version `v0.3.0` or above, try to remove the `console=ttyS0` parameter when booting. You can press `e` to edit the GRUB menu entry and remove the `console=ttyS0` parameter.
```
Comment (gitlawr) [inline docs/install/usb-install.md:50]: Is this still valid for v1.0?

File: docs/rancher/virtualization-management.md [hunk 4]
```diff
@@ -63,11 +63,11 @@ The following example provides a good explanation of how the multi-tenant featur
 1. Create a VM with one of the images that you have uploaded.
 1. Log in with another user, e.g., `project-readonly`, and this user will only have the read permission of this project.
 
-!!!note
-    A known issue was found that allows the [read-only user to be able to manage API actions](https://github.com/harvester/harvester/issues/1406).
-
 ## Delete Imported Harvester Cluster
 
+!!! note
+    You will need to reset the `cluster-registration-url` to empty on the setting page of the imported Harvester cluster first. A known issue will be addressed via [#1686](https://github.com/harvester/harvester/issues/1686).
```
Comment (gitlawr) [inline docs/rancher/virtualization-management.md:69]: This doesn't seem to be necessary. Removing the cluster from rancher dashboard should be enough

File: docs/install/management-address.md [hunk 5]
```diff
@@ -6,21 +6,28 @@ Description: The Harvester provides a virtual IP as the management address.
...
 
-## Usages
-The management address has two usages.
+![](./assets/iso-installed.png)
 
-- Allow users to access the Harvester UI via `https` protocol.
+## How to get the VIP MAC address
 
-- Used by the other nodes to join the cluster. 
+You can get the VIP MAC address via run the following command on the management node:
```
Comment (nunix) [inline docs/install/management-address.md:18]:
```suggestion
To get the VIP MAC address, you can run the following command on the management node:
```

File: docs/install/management-address.md [hunk 6]
```diff
@@ -6,21 +6,28 @@ Description: The Harvester provides a virtual IP as the management address.
...
 
-## Configure VIP 
-Users can specify the VIP during installation. It can either be configured via DHCP or assigned a static one.
+## Usages
+The management address has two usages.
 
-> Note: In PXE boot, Harvester does not support setting the VIP via DHCP. It will be addressed in the next release.
-> Issue: https://github.com/harvester/harvester/issues/1410
+- Allow users to access the Harvester API/UI via `HTTPS` protocol.
+- Used by the other nodes to join the cluster.
```
Comment (nunix) [inline docs/install/management-address.md:32]:
```suggestion
- Allows the access to the Harvester API/UI via `HTTPS` protocol.
- Is the address the other nodes use to join the cluster. 
```

File: docs/install/usb-install.md (hunk 3 above)
Comment (guangbochen) [reply docs/install/usb-install.md:50]: yep

File: docs/rancher/virtualization-management.md (hunk 4 above)
Comment (guangbochen) [reply docs/rancher/virtualization-management.md:69]: IIRC, the Rancher will not delete the agent node pod on the downstream Harvester cluster, and rebooting the agent node will add the cluster back unexpectedly.

File: docs/rancher/rancher-integration.md [hunk 7]
```diff
@@ -15,15 +15,16 @@ _Available as of v0.3.0_
...
 Users can now import and manage multiple Harvester clusters using the Rancher [Virtualization Management](virtualization-management.md) page and leverage the Rancher [authentication](https://rancher.com/docs/rancher/v2.6/en/admin-settings/authentication/) feature and RBAC control for [multi-tenancy](https://rancher.com/docs/rancher/v2.6/en/admin-settings/rbac/) support.
 
 ![virtualization-management](assets/virtualization-management.png)
 
 ## Deploying Rancher
 
-Previously in Harvester v0.2.0, users had the option to enable the embedded Rancher server. This option has been removed from Harvester `v0.3.0`. 
-
-To use Rancher with Harvester, please install the Rancher server separately from the Harvester. As an option, You can spin up a VM in the Harvester and install the Rancher v2.6.1 or above to try out the integration features.
+To use Rancher with Harvester, please install the Rancher server separately from the Harvester. As an option, You can spin up a VM in the Harvester and install the Rancher v2.6.3 or above to try out the integration features.
```
Comment (nunix) [inline docs/rancher/rancher-integration.md:27]:
```suggestion
To use Rancher with Harvester, please install the Rancher and Harvester in two separated servers. If you want to try out the integration features, you can create a VM in Harvester and install Rancher v2.6.3 or above.
```

File: docs/rancher/rancher-integration.md [hunk 8]
```diff
@@ -35,28 +36,26 @@ To use Rancher with Harvester, please install the Rancher server separately from
 
 ```shell
 # for a quick evaluation, you can run the Rancher server with the following command
-$ sudo docker run -d --restart=unless-stopped -p 80:80 -p 443:443 --privileged rancher/rancher:v2.6.2
+$ sudo docker run -d --restart=unless-stopped -p 80:80 -p 443:443 --privileged rancher/rancher:v2.6.3
 ```
 
 !!! Note
-    For more information about how to deploy the Rancher server, please refer to the Rancher [documentation](https://rancher.com/docs/rancher/v2.6/en/quick-start-guide/deployment/)
+    For more information about how to deploy the Rancher server, please refer to the Rancher [documentation](https://rancher.com/docs/rancher/v2.6/en/quick-start-guide/deployment/).
```
Comment (nunix) [inline docs/rancher/rancher-integration.md:43]:
```suggestion
    For more information on how to deploy the Rancher server, please refer to the Rancher [documentation](https://rancher.com/docs/rancher/v2.6/en/quick-start-guide/deployment/).
```

File: docs/rancher/rancher-integration.md [hunk 9]
```diff
@@ -35,28 +36,26 @@ To use Rancher with Harvester, please install the Rancher server separately from
...
 
 ## Virtualization Management
 
-With Rancher's Virtualization Management, users can now import and manage Harvester clusters. By clicking on one of the clusters, users are able to view and manage the downstream Harvester resources such as VMs, images, volumes, etc. Additionally, Rancher's VM feature has leveraged existing Rancher features such as authentication with various auth providers and multi-tenant support.
+With Rancher's Virtualization Management feature, users can now import and manage Harvester clusters.
```
Comment (nunix) [inline docs/rancher/rancher-integration.md:47]:
```suggestion
With Rancher's Virtualization Management feature, you can now import and manage Harvester clusters. 
```

File: docs/index.md [hunk 10]
```diff
@@ -77,26 +77,29 @@ During the installation, you can either choose to form a new cluster or join the
...
+1. Choose the installation device to which the Harvester cluster will be formatted
+     - Note: you will be promoted to select the [MBR](https://en.wikipedia.org/wiki/Master_boot_record) disk partition when install via the BIOS method, otherwise [GPT](https://en.wikipedia.org/wiki/GUID_Partition_Table) will be used by default.
+   ![iso-install-disk.png](./install/assets/iso-install-disk.png)
 1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bond NIC named `harvester-mgmt`, and the IP address can either be configured via DHCP or by static method.
    ![iso-installed.png](./install/assets/iso-nic-config.gif)
 1. Optional: Configure the DNS servers; use commas as delimiters.
-1. Configure the Virtual IP with which you can use to access the cluster or join other nodes to the cluster.
+1. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster <small>(Note: You will need to configure static MAC-to-IP address mapping on your DHCP server to persist the Virtual IP accordingly)</small>.
```
Comment (nunix) [inline docs/index.md:86]:
```suggestion
1. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster <small>(Note: If your IP address is configured via DHCP, you will need to configure static MAC-to-IP address mapping on your DHCP server in order to have a persistant Virtual IP)</small>.
```

File: docs/install/iso-install.md [hunk 11]
```diff
@@ -25,16 +25,18 @@ Note: This [video](https://youtu.be/97ADieBX6bE) shows a quick overview of the I
...
 1. Choose the installation device that Harvester will be formatted to.
+   - Note: you will be promoted to select the [MBR](https://en.wikipedia.org/wiki/Master_boot_record) disk partition when install via the BIOS method, otherwise [GPT](https://en.wikipedia.org/wiki/GUID_Partition_Table) will be used by default.
+     ![iso-install-disk.png](./assets/iso-install-disk.png)
 1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `harvester-mgmt`, and the IP address can either be configured via DHCP or a static assigned a static one.
    ![iso-installed.png](./assets/iso-nic-config.gif)
 1. (Optional) Configure the DNS servers. Use commas as a delimiter.
-1. Configure the `Virtual IP` which you can use to access the cluster or join the cluster to other nodes.
+1. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster <small>(Note: You will need to configure static MAC-to-IP address mapping on your DHCP server to persist the Virtual IP accordingly)</small>.
```
Comment (nunix) [inline docs/install/iso-install.md:33]:
```suggestion
1. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster <small>(Note: If your IP address is configured via DHCP, you will need to configure static MAC-to-IP address mapping on your DHCP server in order to have a persistant Virtual IP)</small>.
```

File: docs/rancher/rancher-integration.md [hunk 12]
```diff
@@ -35,28 +36,26 @@ To use Rancher with Harvester, please install the Rancher server separately from
...
+    For more information about how to deploy the Rancher server, please refer to the Rancher [documentation](https://rancher.com/docs/rancher/v2.6/en/quick-start-guide/deployment/).
 
 ## Virtualization Management
 
-With Rancher's Virtualization Management, users can now import and manage Harvester clusters. By clicking on one of the clusters, users are able to view and manage the downstream Harvester resources such as VMs, images, volumes, etc. Additionally, Rancher's VM feature has leveraged existing Rancher features such as authentication with various auth providers and multi-tenant support.
+With Rancher's Virtualization Management feature, users can now import and manage Harvester clusters. 
+By clicking on one of the clusters, users are able to view and manage the imported Harvester cluster resources like Hosts, VMs, images, volumes, etc. Additionally, the `Virtualization Management` has leveraged existing Rancher features such as authentication with various auth providers and multi-tenant support.
```
Comment (nunix) [inline docs/rancher/rancher-integration.md:48]:
```suggestion
By clicking on one of the clusters, you are able to view and manage the imported Harvester cluster resources like Hosts, VMs, images, volumes, etc. Additionally, the `Virtualization Management` leverages existing Rancher features such as authentication with various auth providers and multi-tenant support.
```

File: docs/rancher/virtualization-management.md [hunk 13]
```diff
@@ -8,13 +8,13 @@ keywords:
 
 For Harvester v0.3.0 and above, virtualization management with the multi-cluster management feature will be supported using Rancher v2.6.x.
 
-1. First, you will need to install Rancher v2.6.1 or above. For testing purposes, you can spin up a Rancher server using the following `docker run` command: 
+1. Firstly, you will need to spin up a Rancher with v2.6.3 or above of Harvester v1.0.0 management. For testing purposes, you can spin up a Rancher server using the following `docker run` command:
```
Comment (nunix) [inline docs/rancher/virtualization-management.md:11]:
```suggestion
1. As a prerequisite, you will need to create a Rancher server with v2.6.3 or above with Harvester v1.0.0 management. For testing purposes, you can spin up a Rancher server using the following `docker run` command: 
```

File: docs/rancher/virtualization-management.md (hunk 4 above)
Comment (nunix) [inline docs/rancher/virtualization-management.md:69]:
```suggestion
    You need to delete, manually, the `cluster-registration-url` value in the settings page of the imported Harvester cluster first. A known issue is open for addressing this bug: [#1686](https://github.com/harvester/harvester/issues/1686).
```

**Review by nunix — COMMENTED** [review]
Overall rephrasing to make it "active voice"

**Review by nunix — APPROVED** [review]

File: docs/rancher/virtualization-management.md (hunk 4 above)
Comment (gitlawr) [reply docs/rancher/virtualization-management.md]: No, it won't add the cluster back on restart. The cluster agent is in charge of the connection, it does not create clusters.

Comment (guangbochen) [reply docs/rancher/virtualization-management.md]: cool, thanks for the clarification.

**Comment by guangbochen** [comment]
@gitlawr updated, thanks.

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**